src/Forms/EffectKaraoke.cs

```diff
@@ -50,7 +50,7 @@ internal void Initialize(Paragraph paragraph)
             AddToPreview(richTextBoxPreview, paragraph.Text);
             RefreshPreview();
             labelTotalMilliseconds.Text = string.Format("{0:#,##0.000}", paragraph.Duration.TotalMilliseconds / TimeCode.BaseUnit);
-            numericUpDownDelay.Maximum = (int)((paragraph.Duration.TotalMilliseconds - 500) / TimeCode.BaseUnit);
+            numericUpDownDelay.Maximum = (decimal)((paragraph.Duration.TotalMilliseconds - 500) / TimeCode.BaseUnit);

```

Comment:
Casting to `int` will trim floating point. (Found while fixing EffectTypeWritter)